cloudfront-cff-s3-geo-uri-rewrite-cdk-dotnet/example-pattern.json

```diff
@@ -0,0 +1,58 @@
+{
+    "title": "CloudFront to CloudFront Functions with URI Rewrite to S3",
+    "description": "Create a Static Website fronted by CloudFront with CloudFront Functions automatically routing visitors to localized content using CloudFront's Geo Location Headers.",
```

Comment:
The description has a maximum length of 150 characters, please shorten it to fit within this limit

Reply:
Updated.